json/fr.json

```diff
@@ -157,21 +158,21 @@
     "Two Factor Authentication" : "Double authentification",
     "Unauthorized": "Non autorisé",
     "Update Password" : "Mettre à jour le mot de passe",
-    "Update your account's profile information and email address." : "Update your account's profile information and email address.",
+    "Update your account's profile information and email address." : "Modifier le profil associé à votre compte ainsi que votre adresse email.",
     "Use a recovery code": "Utilisez un code de récupération",
     "Use an authentication code": "Utilisez un code d'authentification",
     "Verify Email Address": "Vérification de l'adresse email",
     "Verify Your Email Address": "Vérifiez votre adresse email",
     "We were unable to find a registered user with this email address.": "Nous n'avons pas pu trouver un utilisateur enregistré avec cette adresse e-mail.",
     "We won't ask for your password again for a few hours.": "Nous ne vous demanderons plus votre mot de passe pour quelques heures",
     "When two factor authentication is enabled, you will be prompted for a secure, random token during authentication. You may retrieve this token from your phone's Google Authenticator application." : "When two factor authentication is enabled, you will be prompted for a secure, random token during authentication. You may retrieve this token from your phone's Google Authenticator application.",
-    "Whoops! Something went wrong." : "Oups! Un problème est survenu.",
+    "Whoops! Something went wrong." : "Oups ! Un problème est survenu.",
```

Comment:
Why did you add a space character?

Reply:
In French we must have space character before and after exclamation point (in fact, this is what I learnt and see overall the web, but maybe this is incorrect ?)

Reply:
Got it. I invited the people responsible for the French translation to the code review :)

Reply:
+1 for the space